dev/io.openliberty.jakarta.validation.3.1_fat_tck/publish/servers/ValidationTCKServer/jvm.options

```diff
@@ -0,0 +1,3 @@
+-Dcom.ibm.tools.attach.enable=yes
+--add-exports=java.base/jdk.internal.vm.annotation=ALL-UNNAMED
+--add-opens=java.base/jdk.internal.vm.annotation=ALL-UNNAMED
```

Comment:
I think these JVM options can also be removed from the server.
I believe they would be necessary if the TCK did signature testing as part of a TCK test.
```suggestion

```